src/screens/CreatePost/index.js

```diff
@@ -29,6 +28,8 @@ import CreatePollContainer from './elements/CreatePollContainer';
 import CreatePostInput from '../../components/CreatePostInput';
 import Gap from '../../components/Gap';
 import Header from '../../components/Header';
+import ImageCompressionUtils from '../../utils/image/compress';
+import ImageUtils from '../../utils/image';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New imports have been added for `ImageCompressionUtils` and `ImageUtils`. Make sure these modules exist and are correctly exported in the specified paths.
```diff
+ import ImageCompressionUtils from '../../utils/image/compress';
+ import ImageUtils from '../../utils/image';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->